ZenPacks/zenoss/Microsoft/Windows/migrate/MigrateRRDPaths.py

```diff
@@ -28,10 +28,12 @@ class MigrateRRDPaths(ZenPackMigration):
     version = Version(2, 7, 0)
 
     def migrate(self, pack):
+        # setting the default
+        pack.dmd.windows_using_legacy_rrd_paths = False
         if pack.prevZenPackVersion is None:
             return
         installed_version = pkg_resources.parse_version(pack.prevZenPackVersion)
         # we only want to set this if upgrading directly from 2.5.x
         if installed_version < pkg_resources.parse_version('2.6.0'):
-            LOG.info('Setting zWinUseLegacyRRDPath to True we are upgrading directly from 2.5.x')
-            pack.dmd.Devices.setZenProperty('zWinUseLegacyRRDPath', True)
+            LOG.info('Using legacy RRDPath since we are upgrading directly from 2.5.x')
```

Comment:
This log might not be true. They could be upgrading from something earlier than 2.5.x too.

What is the user supposed to do with this information? If there's no reason for the user to know about this, or anything for the user to do about it, then there may be no need to log it at all.